Mathlib/Data/Finsupp/Pointwise.lean

```diff
@@ -103,14 +103,9 @@ instance pointwiseScalar [Semiring β] : SMul (α → β) (α →₀ β) where
       rw [h, smul_zero])
 #align finsupp.pointwise_scalar Finsupp.pointwiseScalar
 
-@[simp]
-theorem coe_pointwise_smul [Semiring β] (f : α → β) (g : α →₀ β) : FunLike.coe (f • g) = f • g :=
-  rfl
-#align finsupp.coe_pointwise_smul Finsupp.coe_pointwise_smul
-
```

Comment:
Why did this go?

Reply:
The `synTaut` linter reports it's a syntactic tautology.

Reply:
Then there is a coercion missing; the statement should really be `FunLike.coe (f • g) = f • FunLike.coe `, though written with `\u` isntead of `FunLike.coe` to allow your nondependent instance to fire.